go.mod

```diff
@@ -5,6 +5,7 @@ go 1.13
 require (
 	github.com/cosmos/cosmos-sdk v0.38.1
 	github.com/gorilla/mux v1.7.3
+	github.com/pkg/errors v0.9.1
```

Comment:
why did you add this?

Reply:
`go mod` discovered it in here:   https://github.com/enigmampc/EnigmaBlockchain/blob/6cd448d376da812de940b7f20c870ad64f52c398/cosmwasm/lib/incubator-teaclave-sgx-sdk/samplecode/ue-ra/ue-ra-client-go/cert.go#L10

So my IDE auto-added it to `go.mod`